tests/test_filter_jit.py

```diff
@@ -204,3 +204,51 @@ def __call__(self, x):
     eqx.filter_jit(m)(y)
     eqx.filter_jit(m)(y)
     assert num_traces == 1
+
+
+@pytest.fixture
+def log_compiles_config():
+    """Setup and teardown of jax_log_compiles flag"""
+    jax.config.update('jax_log_compiles', True)
+    yield
+    jax.config.update('jax_log_compiles', False)
```

Comment:
Can this be set to reset `jax_log_compiles` to whatever it was originally, and not necessarily just `False`?

Reply:
Sure, I'll change that. There's a context manager for all these flags, apparently.